docker/.env.local

```diff
@@ -0,0 +1,6 @@
+ELASTICSTACK_VERSION="8.0.0"
+USER_ELASTIC_PASSWORD="huml1b_el4st1c"
+USER_KIBANA_SYSTEM_PASSWORD="huml1b_k1b4n4"
+
+# Increase or decrease based on the available host memory (in bytes)
+ELASTICSTACK_MEM_LIMIT="1073741824"
```

Comment:
Can you mention this in README, please? That you can increase this locally if your PC can handle it?